ecosystem.config.js

```diff
@@ -99,12 +93,9 @@ module.exports = {
         ...process.env,
         PROJECT_NAME,
         DP_SHARD: "1",
-        TP_DEGREE: "2",
-        CUDA_VISIBLE_DEVICES: "4,5",
-        MAX_CATCHUP_WINDOWS: "5",
-        TMPDIR: "/ephemeral/tmp",
-        TEMP: "/ephemeral/tmp",
-        TMP: "/ephemeral/tmp"
+        TP_DEGREE: "1",
+        CUDA_VISIBLE_DEVICES: "0",
+        MAX_CATCHUP_WINDOWS: "5"
```

Comment:
I missed this. I am not sure about this one either. I think it makes sense for testing but outside of tests it we don't want it really